source/WindowScroller/WindowScroller.example.js

```diff
@@ -0,0 +1,76 @@
+/** @flow */
+import Immutable from 'immutable'
+import React, { Component, PropTypes } from 'react'
+import { ContentBox, ContentBoxHeader, ContentBoxParagraph } from '../demo/ContentBox'
+import WindowScroller from './WindowScroller'
+import VirtualScroll from '../VirtualScroll'
+import shallowCompare from 'react-addons-shallow-compare'
+import styles from './WindowScroller.example.css'
+
+export default class AutoSizerExample extends Component {
+  static propTypes = {
+    list: PropTypes.instanceOf(Immutable.List).isRequired
+  }
+
+  constructor (props) {
+    super(props)
+    this._rowRenderer = this._rowRenderer.bind(this)
+  }
+
+  render () {
+    const { list, ...props } = this.props
+
+    return (
+      <ContentBox
+        {...props}
+      >
+        <ContentBoxHeader
+          text='WindowScroller'
+          sourceLink='https://github.com/bvaughn/react-virtualized/blob/master/source/WindowScroller/WindowScroller.example.js'
+          docsLink='https://github.com/bvaughn/react-virtualized/blob/master/docs/WindowScroller.md'
+        />
+
+        <ContentBoxParagraph>
+          This component decorates <code>VirtualScroll</code>, <code>FlexTable</code>, or any other component
+          and manages the window scroll to scroll through the list
+        </ContentBoxParagraph>
+
+        <div className={styles.WindowScrollerWrapper}>
+          <WindowScroller>
+            {({ height, scrollTop }) => (
+              <VirtualScroll
+                autoHeight
+                className={styles.VirtualScroll}
+                height={height}
+                rowCount={list.size}
+                rowHeight={30}
+                rowRenderer={this._rowRenderer}
+                scrollTop={scrollTop}
+                width={460}
```

Comment:
This will cause it to overflow on narrow browsers (eg iPhone). Suggestion: Use `AutoSizer` here too?

```diff
diff --git a/source/WindowScroller/WindowScroller.example.js b/source/WindowScroller/WindowScroller.example.js
index 1c0a8ca..8901b18 100644
--- a/source/WindowScroller/WindowScroller.example.js
+++ b/source/WindowScroller/WindowScroller.example.js
@@ -2,6 +2,7 @@
 import Immutable from 'immutable'
 import React, { Component, PropTypes } from 'react'
 import { ContentBox, ContentBoxHeader, ContentBoxParagraph } from '../demo/ContentBox'
+import AutoSizer from '../AutoSizer'
 import WindowScroller from './WindowScroller'
 import VirtualScroll from '../VirtualScroll'
 import shallowCompare from 'react-addons-shallow-compare'
@@ -38,16 +39,20 @@ export default class AutoSizerExample extends Component {
         <div className={styles.WindowScrollerWrapper}>
           <WindowScroller>
             {({ height, scrollTop }) => (
-              <VirtualScroll
-                autoHeight
-                className={styles.VirtualScroll}
-                height={height}
-                rowCount={list.size}
-                rowHeight={30}
-                rowRenderer={this._rowRenderer}
-                scrollTop={scrollTop}
-                width={460}
-              />
+              <AutoSizer disableHeight>
+                {({ width }) => (
+                  <VirtualScroll
+                    autoHeight
+                    className={styles.VirtualScroll}
+                    height={height}
+                    rowCount={list.size}
+                    rowHeight={30}
+                    rowRenderer={this._rowRenderer}
+                    scrollTop={scrollTop}
+                    width={width}
+                  />
+                )}
+              </AutoSizer>
             )}
           </WindowScroller>
         </div>
```